modules/application-health-about.adoc

```diff
@@ -9,121 +9,131 @@ A probe is a Kubernetes action that periodically performs diagnostics on a
 readiness probe can be used to signal to the endpoints controller that even
 though a container is running, it should not receive any traffic from a proxy.
-
++
 For example, a Readiness check can control which Pods are used. When a Pod is not ready,
 it is removed.
 
-Liveness Probe::
+Liveness check::
 A Liveness checks determines if the container in which it is scheduled is still
-running. If the liveness probe fails due to a condition such as a deadlock, the kubelet kills the container The container then
+running. If the liveness probe fails due to a condition such as a deadlock, the kubelet kills the container. The container then
 responds based on its restart policy.
-
-For example, a liveness probe on a node with a `restartPolicy` of `Always` or `OnFailure`
++
+For example, a Liveness check on a node with a `restartPolicy` of `Always` or `OnFailure`
 kills and restarts the Container on the node.
 
-.Sample Liveness Check
+Startup check::
+Legacy applications can require additional startup time on their first initialization. This situation can make it difficult to set up liveness probe parameters without compromising the fast response to deadlocks that a liveness probe provides. To prevent containers from starting slowly, configure a startup probe using the same command, HTTP or TCP check, with `failureThreshold` and `periodSeconds` values long enough to handle the worst case startup time.
++
+For example, you can add a Startup check with a maximum of 5 minutes (30 * 10 = 300s) to a Liveness check. After the Startup check succeeds the first time, the liveness probe takes over to provide a fast response to container deadlocks. If the startup probe never succeeds within the specified time period, the container is killed and is subject to the pod `restartPolicy`.
+
+Types of probes::
+Liveness, Readiness, and Startup checks can be configured in three ways:
++
+--
+* HTTP probe. When using an HTTP probe, the kubelet determines the healthiness of the container by using a web hook. This probe is successful if the HTTP response code is between 200 and 399.
++
+You can use an HTTP probe with applications that return HTTP status codes when completely initialized.
+
+* Container Execution probe. When using a Container Execution probe, the kubelet executes a command inside the container. The probe is successful if the probe exits with
+a `0` status.
+
+* TCP Socket probe. When using a TCP Socket probe, the kubelet attempts to open a socket to the container. The container is only
+considered healthy if the probe can establish a connection. You can use a TCP socket probe with applications that do not start listening until
+initialization is complete.
+--
++
+--
+Probes have a number of fields that you can use to more precisely control the behavior of Liveness and Readiness checks:
+* `initialDelaySeconds`: The number of seconds after the container has started before the probe starts. The defaults is `0`.
```

Comment:
Even if the initialDelaySeconds is greater or lower than the periodSeconds, the first probe will be scheduled in a timeframe between the **initialDelaySeconds** and the **initialDelaySeconds + periodSeconds**.
I think that you should introduce the notion of scheduling instead to say that the "probe starts".
I will more likely suggest something like:
> The number of seconds after the container has started before the probe can be scheduled.

which implecitely suggest that the probe can take longer to start

Reply:
@vlours Thanks for the suggestion. I made the change.